### ISSUE TITLE
Errors: browserVersion cannot  be empty and no match capabilities found.

### ISSUE THREAD

**Issue by pilotstew** [stevepryde/thirtyfour#8]
I am trying to build a small web scaper while learning rust. However, I am unable to connect to chromedriver. 

Arch Linux
rustc 1.44.0-nightly 
ChromeBrowser Version: **83.0.4103.61**
Chromedriver Version: **83.0.4103.39**

Here is my original code:
```
let caps = DesiredCapabilities::chrome();
let driver = WebDriver::new("http://localhost:4444", &caps)
    .await
    .expect("could not connect to webdriver");
```
This results in a invalid argument: cannot parse capability error the Rust DEBUG logs:
```
thread 'main' panicked at 'could not connect to webdriver: InvalidArgument(WebDriverErrorInfo { status: 400, error: "", value: WebDriverErrorValue { message: "invalid argument: cannot parse capability: browserVersion\nfrom invalid argument: cannot be empty", error: Some("invalid argument"), stacktrace: Some("#0 0x5646c0b6c579 <unknown>\n"), data: None } })', src/main.rs
```
And the chromedriver logs:
```
[INFO]: RESPONSE InitSession ERROR invalid argument: cannot parse capability:
 browserVersion from invalid argument: cannot be empty
[DEBUG]: Log type 'driver' lost 1 entries on destruction
[INFO]: COMMAND InitSession {
   "capabilities": {
      "alwaysMatch": {
         "browserName": "chrome",
         "browserVersion": "",
         "platformName": "ANY"
      },
      "firstMatch": [ {

      } ]
   },
   "desiredCapabilities": {
      "browserName": "chrome",
      "platform": "ANY",
      "version": ""
   }
}
[INFO]: RESPONSE InitSession ERROR invalid argument: cannot parse capability: 
browserVersion from invalid argument: cannot be empty
[DEBUG]: Log type 'driver' lost 1 entries on destruction
```
So I tried adding the browserVersion manually:
```
let mut caps = DesiredCapabilities::chrome();
caps.set_version("83.0.4103.61")
    .expect("failed to set browserVersion for webdriver");
let driver = WebDriver::new("http://localhost:4444", &caps)
    .await
    .expect("could not connect to webdriver");
```
But I get a invalid argument: cannot be empty error in Rust DEBUG logs:
```
thread 'main' panicked at 'could not connect to webdriver: InvalidArgument(WebDriverErrorInfo { status: 400, error: "", value: WebDriverErrorValue { message: "invalid argument: cannot parse capability: browserVersion\nfrom invalid argument: cannot be empty", error: Some("invalid argument"), stacktrace: Some("#0 0x5646c0b6c579 <unknown>\n"), data: None } })', src/main.rs
```
And a "No matching capabilities found" error in the chromedriver logs:
```
[INFO]: RESPONSE InitSession ERROR session not created: No matching capabilities found
[DEBUG]: Log type 'driver' lost 1 entries on destruction
[INFO]: COMMAND InitSession {
   "capabilities": {
      "alwaysMatch": {
         "browserName": "chrome",
         "browserVersion": "83.0.4103.61",
         "platformName": "ANY"
      },
      "firstMatch": [ {

      } ]
   },
   "desiredCapabilities": {
      "browserName": "chrome",
      "platform": "ANY",
      "version": "83.0.4103.61"
   }
}
[INFO]: RESPONSE InitSession ERROR session not created: No matching capabilities found
[DEBUG]: Log type 'driver' lost 1 entries on destruction
```
What am I doing wrong here?

**Comment by stevepryde** [comment]
I originally created the desiredCapabilities to work with selenium, but it seems chromedriver does not like the platform being specified as "ANY". Removing that should fix it.

I've published an update (v0.11.0) to fix this. Let me know if it works for you.

**Comment by pilotstew** [comment]
Wonderful, that solves it. Thank You
